reactioncommerce/reaction
# Fix: Not being able to scroll after using search function #3314

This is a fix for issue #3314.

**Fix**

You can now scroll as normal after searching for a product or exiting the search function. This involved removing the the 'overflow: hidden' style tag from the body HTML element after a search is complete.

**Test**

1. Click on the search icon
2. Search for a product
3. Click on the product
4. Observe that you can now scroll as intended on the page.
